usl_pipeline/usl_lib/usl_lib/storage/metastore.py

```diff
@@ -266,6 +266,29 @@ def update_scaling_done(
             {"needs_scaling": False, "feature_matrix_path": scaled_feature_matrix_path},
         )
 
+    @staticmethod
+    def delete_all_for_study_area(
+        db: firestore.Client,
+        study_area_name: str,
+        page_size: Optional[int] = None,
+    ) -> None:
+        """Deletes all the chunks from chunk sub-collection for a given study area.
+
+        Args:
+            db: The firestore database client to use.
+            study_area_name: The study area to delete.
+            page_size: Optional number of documents in the processing page.
+        """
+        docs = (
+            db.collection(STUDY_AREAS)
+            .document(study_area_name)
+            .collection(STUDY_AREA_CHUNKS)
+            .list_documents(page_size=page_size)
+        )
+        for doc in docs:
+            print(f"Deleting chunk {doc.get().to_dict()}")
```

Comment:
Is this meant to be a `logging.info` statement? Or a leftover debug line?

Reply:
Sorry, forgot to deleted before committing.